### PR TITLE
RC: Updates for VPC peering UI & multiple CIDRs

### PR DESCRIPTION
**Pull request by rrelledge**
Updates include:
- New instructions, screenshots, and buttons
- Separate instructions for AWS and GCP
- Multiple CIDRs for VPC peering
- Consolidated duplicated content from network security doc and added relevant links instead

Staged previews:
[VPC peering](https://docs.redis.com/staging/release-rc-vpc/rc/security/vpc-peering/)
[Network security > Virtual private clouds](https://docs.redis.com/staging/release-rc-vpc/rc/security/database-security/network-security/#virtual-private-clouds)

**Reminder**: Don't merge this PR until the scheduled date.

### CODE REVIEW

File: content/rc/security/vpc-peering.md [hunk 1]
```diff
@@ -10,67 +10,124 @@ aliases:
 
 A [VPC peering connection](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) is a networking connection between two virtual PCs that uses private IP addresses to route traffic between them.  Instances in either VPC can communicate with each other as if they are within the same network.
```
Comment (joniredis) [inline content/rc/security/vpc-peering.md:11]: @rrelledge this link points to AWS docs but it is part of the cloud agnostic section of this doc. Is that on purpose or an oversight?

**Comment by joniredis** [comment]
Thanks @rrelledge! I think that the VPC peering page is excellent. 

Regarding the network security IMHO the difference between [two types of network security: IP Restrictions and VPCs](https://docs.redis.com/staging/release-rc-vpc/rc/security/database-security/network-security/#virtual-private-clouds) would be clearer if we would say something like:

> two types of network security: database-level CIDR allow lists and VPC-wide CIDR allow lists


Also it would be great if we could replace the screenshots of the network security page to those of the new UI (not sure if that is planned for a future effort).

File: content/rc/security/database-security/network-security.md [hunk 2]
```diff
@@ -68,58 +68,15 @@ To restrict a database to a specific set of source IP addresses or subnets:
...
+Databases in Flexible and Annual subscriptions are almost always deployed in a Redis VPC. In most cases, you'll need to create a [VPC peering connection]({{<relref "/rc/security/vpc-peering">}}) to access these databases. A VPC peering connection allows unrestricted network access between two VPCs.
 
 How you create these connections and the features supported vary somewhat by public cloud provider. You can read about VPC usage for [AWS](#vpcs-with-aws), [GCP](#vpcs-with-gcp), and [Azure](#vpcs-with-azure) below.
 
 ### VPCs with AWS
 
-Subscriptions that run on AWS support two VPC options. To ensure that that you can securely connect to your database, you must either [create a VPC peering connection](#creating-a-vpc-peering-connection) or [deploy your subscription in your own VPC](#deploying-in-your-own-vpc).
+Subscriptions that run on AWS support two VPC options. To ensure that that you can securely connect to your database, you need to either [create a VPC peering connection]({{<relref "/rc/security/vpc-peering#aws-vpc-peering">}}) or [deploy your subscription in your own VPC](#deploy-in-your-own-vpc).
```
Comment (joniredis) [inline content/rc/security/database-security/network-security.md:77]: @rrelledge regarding "deploy your subscription in your own VPC" this is something that today is available only to a few veteran customers and not enabled by default. Not sure if we should mention this still in the docs.

**Comment by joniredis** [comment]
@rrelledge Also regarding the [deploy-in-your-own-vpc](https://docs.redis.com/staging/release-rc-vpc/rc/security/database-security/network-security/#deploy-in-your-own-vpc) 

> When the Create Custom Subscription screen appears, locate the Networking section of the Setup tab and then select the option to deploy in an existing VPC.

The above is only true for customers that have a cloud account set and during subscription creation they select one. Note this is only available to a select few veteran customers and not available by default to new customers (it is a feature flag that the customer success need to enable for them).

File: content/rc/security/vpc-peering.md (hunk 1 above)
Comment (rrelledge) [reply content/rc/security/vpc-peering.md:11]: Good catch. I can take a look and see what other options might exist. A quick web search doesn't return any obvious agnostic resources for this though.

File: content/rc/security/database-security/network-security.md (hunk 2 above)
Comment (rrelledge) [reply content/rc/security/database-security/network-security.md:77]: If you think the "deploy your subscription in your own VPC" section is no longer needed, I can go ahead and remove it.

**Comment by rrelledge** [comment]
> Also it would be great if we could replace the screenshots of the network security page to those of the new UI (not sure if that is planned for a future effort).

@joniredis Updating the rest of the network security section was originally planned to be a separate effort, but since I've already changed so many things for that doc in this PR, I think I'll just go ahead and include the rest of these changes here too.

**Comment by rrelledge** [comment]
**Reminder:** Don't merge this PR until the scheduled date.

**Review by lanceleonard — APPROVED** [review]

**Pull request merged**